frontend/src/components/common/RadioGroup/RadioGroup.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div class="row">
+        <b-form-radio-group
+            id="radio-group-1"
```

Comment:
I wonder if we need "id" and "name" here, do they need to be unique?

Reply:
They are not needed, removing those. Thanks for spotting it.